### DISCUSSION TITLE
Isp re-programming  on soldered attiny167

### DISCUSSION THREAD

**Discussion by Rimbaldo** [SpenceKonde/ATTinyCore#552] [Support / Q & A]
Is it ok to reprogram a soldered  attiny167  on a pcb, by using isp, when one of the pins needed (MISO) has a 10K resistor pulled to ground? The others pins, besides 5V and GND are left floating.

**Comment by SpenceKonde** [comment]
Yes that's fine, for sure - yout part will be able to overcome the 10k pulldown to drive the MISO line easily.
